### PR TITLE
#105 issue - Added Sinhala language

### PR DESCRIPTION
**Pull request by STEdirisuriya**
added Sinhala (Sri Lanka) language translation
https://github.com/fossasia/publiccode.asia/issues/105

### CODE REVIEW

**Comment by STEdirisuriya** [comment]
@Remorax  How to fix travis errors

**Comment by STEdirisuriya** [comment]
Please give me some time I am working

**Comment by STEdirisuriya** [comment]
@Remorax It's Ok Now

File: site/languages/strings.si.toml [hunk 1]
```diff
@@ -0,0 +1,126 @@
...
+[Languages.si.arguments]
+  headline    = "පොදු කේතයට හේතු"
+  followup    = "නිදහස් හා විවෘත මූලාශ්‍ර මෘදුකාංග ප්‍රසිද්ධියේ මූල්‍යනය කළ මෘදුකාංග සඳහා පෙරනිමි විකල්පයක් විය යුතු බව ඔබ විශ්වාස කරනවාද? **ඔබගේ දේශපාලන ආයාචනා ඒත්තු ගන්වමු!**"
+  buttonText  = "විවෘත කේතනය ආරම්භයට අත්සන් කරන්න"
+[[Languages.si.arguments.list]]
+  icon        = "fa-usd"   <-- do not translate this
+  title       = "බදු ඉතිරි කිරීම"
+  description = "සමාන යෙදුම් සෑම විටම මුලසිට වැඩසටහන් ගත කළ යුතු නැත."
+[[Languages.si.arguments.list]]
+  icon        = "fa-handshake-o"   <-- do not translate this
```
Comment (abishekvashok) [inline site/languages/strings.si.toml]: You need a # before <--
In toml # is used to begin a comment.

**Comment by STEdirisuriya** [comment]
@isuruAb I did

File: site/languages/strings.si.toml [hunk 2]
```diff
@@ -0,0 +1,126 @@
...
+  buttonText  = "වැඩි වාසි"
+
+
+#Arguments section
+[Languages.si.arguments]
+  headline    = "පොදු කේතයට හේතු"
+  followup    = "නිදහස් හා විවෘත මූලාශ්‍ර මෘදුකාංග ප්‍රසිද්ධියේ මූල්‍යනය කළ මෘදුකාංග සඳහා පෙරනිමි විකල්පයක් විය යුතු බව ඔබ විශ්වාස කරනවාද? **ඔබගේ දේශපාලන ආයාචනා ඒත්තු ගන්වමු!**"
+  buttonText  = "විවෘත කේතනය ආරම්භයට අත්සන් කරන්න"
+[[Languages.si.arguments.list]]
+  icon        = "fa-usd"   <-- do not translate this
```
Comment (akshatnitd) [inline site/languages/strings.si.toml]: Replace `<--` with '#'

File: site/languages/strings.si.toml [hunk 3]
```diff
@@ -0,0 +1,126 @@
...
+  title       = "බදු ඉතිරි කිරීම"
+  description = "සමාන යෙදුම් සෑම විටම මුලසිට වැඩසටහන් ගත කළ යුතු නැත."
+[[Languages.si.arguments.list]]
+  icon        = "fa-handshake-o"   #<-- do not translate this
+  title       = "එක්වන්න"
+  description = "ප්‍රධාන ව්යාපෘති සඳහා දරන ප්‍රයත්නයන්ගෙන් විශේෂඥ ඥාණය හා පිරිවැය බෙදාගත හැකිය."
+[[Languages.si.arguments.list]]
+  icon        = "fa-users"   <-- do not translate this
```
Comment (akshatnitd) [inline site/languages/strings.si.toml]: Replace `<--` with '#'

File: site/languages/strings.si.toml [hunk 4]
```diff
@@ -0,0 +1,126 @@
...
+  title       = "මහජනතාවට සේවය කිරීම"
+  description = "මහජනතාව විසින් ගෙවන ලද යෙදුම් සෑම අයෙකුටම ලබා ගත හැකි විය යුතුය."
+[[Languages.si.arguments.list]]
+  icon        = "fa-lightbulb-o"   <-- do not translate this
```
Comment (akshatnitd) [inline site/languages/strings.si.toml]: Replace `<--` with '#'

**Comment by mariobehling** [comment]
Travis is failing. Please check.

Please also ensure all pages are available in new languages.

**Comment by hpdang** [comment]
Thanks @STEdirisuriya

**Pull request merged**